include/nds/arm9/math.h

```diff
@@ -400,6 +404,8 @@ static inline uint32_t sqrt64(uint64_t a)
 ///
 /// @warning
 ///     Not safe to call inside an interrupt handler.
+/// @warning
+///     Cannot be used concurrently with integer square root functions.
```

Comment:
Does this warning really matter? If anything, we should have it at the top of the file, not in one function (and leave the rest unchanged).

Reply:
The idea was to highlight a behavior difference to normal sqrtf. Someone might assume this is some extra function instead of repurposing the existing functions.

Reply:
Yeah, but the div functions have the same issue, right? That's why I mentioned that a note on top of the file would be nice:

```
/// @brief hardware coprocessor math instructions.
///
/// Warning: there is only one divider and one square root unit [...]
```

Reply:
I added a warning to the top. Let me know if it looks right in doxygen.